web/src/pages/Cases/CaseDetails/Overview/Policies.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { landscapeStyle } from "styles/landscapeStyle";
+import { IPFS_GATEWAY } from "consts/index";
+import PolicyIcon from "svgs/icons/policy.svg";
+
+const ShadeArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  padding: calc(16px + (20 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875)
+    calc(16px + (32 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+  margin-top: 16px;
+  background-color: ${({ theme }) => theme.mediumBlue};
+  > p {
+    font-size: 14px;
+    margin-top: 0;
+    margin-bottom: 16px;
+    color: ${({ theme }) => theme.primaryBlue};
+    ${landscapeStyle(
+      () => css`
+        margin-bottom: 0;
+      `
+    )};
+  }
+
+  ${landscapeStyle(
+    () => css`
+      flex-direction: row;
+      justify-content: space-between;
+    `
+  )};
+`;
+
+const StyledA = styled.a`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  > svg {
+    width: 16px;
+    fill: ${({ theme }) => theme.primaryBlue};
+  }
+`;
+
+const LinkContainer = styled.div`
+  display: flex;
+  gap: calc(8px + (24 - 8) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+`;
+
+interface IPolicies {
+  disputePolicyURI?: string;
+  courtId?: number;
+}
+
+export const Policies: React.FC<IPolicies> = ({ disputePolicyURI, courtId }) => {
+  return (
+    <ShadeArea>
+      <p>Make sure you read and understand the Policies</p>
+      <LinkContainer>
+        {disputePolicyURI && (
```

Comment:
same as with the previous cases, let's see how does it look using the ternary operator